traitsui/qt4/range_editor.py

```diff
@@ -449,6 +449,9 @@ def update_object_on_scroll(self, pos):
     def update_object_on_enter(self):
         """ Handles the user pressing the Enter key in the text field.
         """
+        # it is possible we get the event after the control has gone away
+        if self.control is None:
+            return
```

Comment:
@kitchoi This isn't really directly a part of this PR.  However, this check was included in the update_object_on_enter method for the SimpleSliderEditor, but not on LargeRangeSliderEditor for Idk what reason.  When running the tests I was seeing a failure where this was called with self.control as None.  I can't see why this check would ever be a problem (seeing as it is used in the same way in the other editor style).  If needed I can undo this change, and remove the test which fails because of it (or I could make the change in a separate PR).  Let me know if this was reasonable to change or not.

Reply:
This change looks reasonable to me... persumably this was motivated by a failure of a test you added?

Reply:
Note: the same thing was added for wx